### ISSUE TITLE
Can't control Princess Smart Infrared Panel Heater 540

### ISSUE THREAD

**Issue by m3tc0n** [dennis1804/homewizard-climate-hass#18]
I have a Princess Smart IR Panel Heater 540w (model no. 01.343540.01.001). I have added this panel to the integration and it is also found. But then I cannot control the panel. Is there a solution for this? Thanks in advance.

**Comment by m3tc0n** [comment]
Anyone?

**Comment by m3tc0n** [comment]
I don't think it's because of the integration, as I now also see in the Homewizard Climate app that there is no connection to the infrared panel.

(I've already tried several things, such as deleting the panel and adding it to the app again, but I can't get it to work. No connection can be made to the panel...)

**Comment by dennis1804** [comment]
oh great to see you got the issues too, i got an error saying there was no active subscription for the device and i worried i broke my IR panel.

**Comment by m3tc0n** [comment]
I'm now getting this error: 

![Schermafbeelding 2023-12-04 134914](https://github.com/dennis1804/homewizard-climate-hass/assets/106514117/d5ac1366-eefa-45ba-b46b-d90be2e71e40)

**Comment by dennis1804** [comment]
@m3tc0n, yes that's the same error i got. i reverted the changes this morning because of the issues, didn't want to brick other devices.
luckily it was at homewizards side so i just changed it back to the fix.
please check if you can get the latest hash (f4e371d) through hacs, that one should contain the fix.

**Comment by m3tc0n** [comment]
OK, I will try that. Thanks
